sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -670,7 +760,7 @@ export class ContainerResponse extends ResourceResponse<ContainerDefinition & Re
 
 // @public
 export class Containers {
-    constructor(database: Database, clientContext: ClientContext);
+    constructor(database: Database, clientContext: ClientContext, encryptionManager?: EncryptionManager);
```

Comment:
This constructor needs to be marked hidden/internal. Users cannot instantiate this class as clientContext is internal class.

Reply:
done